### PR TITLE
chore: add dmt lint

### PR DESCRIPTION
**Pull request by KraMorK**
## Description

Current matrix tests do not cover all needs
## Why do we need it, and what problem does it solve?

For matrix tests an alternative solution is used
## Why do we need it in the patch release (if we do)?

Not related to release
## What is the expected result?

An alternative solution is used for tests
## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: ci
type: chore
summary: For matrix tests an alternative solution is used.
impact_level: low
```

### CODE REVIEW

File: werf.yaml [hunk 1]
```diff
@@ -262,17 +262,11 @@ shell:
...
-  beforeInstall:
-  - apk add --no-cache git openssh-client
-  - mkdir -p ~/.ssh && echo "StrictHostKeyChecking accept-new" > ~/.ssh/config
   install:
-  # Use v1.60.1 for go1.23 - https://github.com/golangci/golangci-lint/issues/4837
-  - export GOPROXY={{ .GOPROXY }}
-  - git clone --depth 1 {{ .SOURCE_REPO }}/golangci/golangci-lint --branch v1.60.1
-  - cd golangci-lint/
-  - CGO_ENABLED=0 GOOS=linux go build -ldflags '-s -w -extldflags "-static"' -o /usr/local/bin/golangci-lint cmd/golangci-lint/main.go
+  - curl -sL https://github.com/golangci/golangci-lint/releases/download/v1.60.1/golangci-lint-1.60.1-linux-amd64.tar.gz | tar -zx --strip-components 1 -C /tmp
```
Comment (name212) [inline werf.yaml:268]: why do we change to download from github?

It did for sameness with cse and oss

**Review by name212 — CHANGES_REQUESTED** [review]
one thing

File: werf.yaml (hunk 1 above)
Comment (KraMorK) [reply werf.yaml:268]: I will reply with a private message

Comment (KraMorK) [reply werf.yaml:268]: We decided to take utilities that do not participate in the final images ready-made

**Review by name212 — APPROVED** [review]

**Review by yalosev — APPROVED** [review]

**Pull request merged**